src/clj/game/core/payment.clj

```diff
@@ -96,9 +96,8 @@
                        (payable? % state side eid card))
                  costs)
        costs
-       (when title
-         (toast state side (str "Unable to pay for " title "."))
-         false)))))
+       (do (when title (toast state side (str "Unable to pay for " title ".")))
```

Comment:
I changed it to match the doc more literally - is it idiomatic in clojure to treat falsy uniformly? The other thing that threw me off here, was curious about the existing false, but taking a closer look I better understand since toast doesn't return a falsy value. Happy to revert it

Reply:
Nah, on second thought, you don't have to revert it. If the tests pass, this is perfectly fine. For more information about "falsey" values, check out [the guide](https://clojure.org/guides/learn/flow#_truth)